### ISSUE TITLE
encrypt .sqlite files and anything else that might contain privacy sensitive data using passphrase

### ISSUE THREAD

**Issue by thedavidmeister** [byteball/byteball#168]
Currently the passphrase (if actually set https://github.com/byteball/byteball/issues/167) is used to encrypt the private key.

However, even ignoring the private key, there seems to be data contained in the .sqlite files that can reveal potentially private or identifying details about the owner of the wallet.

Ideally, if an attacker can get a copy of the byteball data files but does not have the passphrase for the wallet/private key, there is no other sensitive or potentially deanonymizing data revealed to the attacker.

Obviously, the wallet would need to decrypt the db before use, providing an attack opportunity, but the security of the db "at rest" would be improved - realistically most users have the wallet app open only a tiny percentage of the time.

**Comment by ByteFan** [comment]
+1

**Comment by tonyofbyteball** [comment]
Fair enough, there are encryption options in sqlite that need to be explored.
